orderer/common/follower/block_puller.go

```diff
@@ -153,18 +177,21 @@ func (creator *BlockPullerCreator) BlockFetcher(configBlock *common.Block, stopC
 		return nil, err
 	}
 
+	var verifyBlockFunc protoutil.BlockVerifierFunc
 	if bftEnabled && f > 0 {
 		fc.MaxByzantineNodes = f
+		// setting verifyBlockFunc, will make BlockFetcher pull genesis block in a bft setting and
+		// intialize the VerifyBlock function
+		verifyBlockFunc = nil
+	} else {
+		verifyBlockFunc = verifierFactory(configBlock)
```

Comment:
How does this work in the case of CFT? What makes you be able to verify the blocks in this case?

Reply:
I depend on the verifier created by verifierFactory

Reply:
If i keep verifyBlockFunc==nil, in cft, it would pull genesis block from multiple orderers, which seems like bft case, so I separated them into two cases.